omnigan/losses.py

```diff
@@ -270,17 +273,15 @@ def get_losses(opts, verbose):
         loss_classifier = MSELoss()
     else:
         loss_classifier = CrossEntropy()
-    losses["G"]["classifier"] = loss_classifier 
+    losses["G"]["classifier"] = loss_classifier
     # -------------------------------
     # -----  Classifier Losses  -----
     # -------------------------------
-    losses["C"] = loss_classifier   
+    losses["C"] = loss_classifier
     # ----------------------------------
     # -----  Discriminator Losses  -----
     # ----------------------------------
     losses["D"] = GANLoss(
-        soft_shift=opts.dis.soft_shift,
-        flip_prob=opts.dis.flip_prob,
-        verbose=verbose,
-    )   
+        soft_shift=opts.dis.soft_shift, flip_prob=opts.dis.flip_prob, verbose=verbose,
+    )
```

Comment:
@alexrey88 can you set up flake8 and black (with max line length 88)? if <- does not make sense let's chat on slack

Reply:
I already have both, with --line-length==88 for black, and when I hit save it divides the line in 3.. :/